_rules/SC1-1-1-image-accessible-name-is-placeholder-text.md

```diff
@@ -16,11 +16,11 @@ authors:
 
 ### Applicability
 
-The rule applies to HTML `input type="image" `and `area` elements or any HTML element with the semantic role of `img` [exposed to assistive technologies](#exposed-to-assistive-technologies), where the [accessible name](#accessible-name) exclusively uses [generic placeholder text](#generic-placeholder-text).
+The rule applies to HTML `input type="image"` and `area` elements, or any HTML element with the semantic role of `img` [exposed to assistive technologies](#exposed-to-assistive-technologies), where the [accessible name](#accessible-name) exclusively uses [generic placeholder text](#generic-placeholder-text).
```

Comment:
Are `input type="image"` and `area` elements only applicable if they're exposed to AT? If so, the sentence should be restructured to reflect this:

> The rule applies to any HTML `input type="image"` or `area` element, or any HTML element with a [semantic role](#semantic-role) of `img`, where the element is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an [accessible name](#accessible-name) that exclusively uses [generic placeholder text](#generic-placeholder-text).